### ISSUE TITLE
Could you please provide the indices of the selected data?

### ISSUE THREAD

**Issue by jcjdataselection** [G-JWLee/COINCIDE_code#10]
Hello! Could you please provide the indices of the selected data? I have successfully run the program, but the chosen data does not yield satisfactory results. In my experiment, approximately 27,000 samples were selected from GQA, 13,000 from OCR, 9,000 from VG, and 2,900 from TextVQA.

**Comment by G-JWLee** [comment]
Hi! Thanks for your interest in our work.

Based on previous issue reports, the result could be different depending on the FAISS version.

BTW, as analysed in our paper's appendix, we observed that samples from VQA tasks (e.g., VQAv2, GQA, OKVQA ...) were majorly selected.

If you let us know your email, we would like to send you the indices for your reference. Please understand that we only have the indices of a 20% pruning ratio for the LLaVA instruction tuning dataset.

**Comment by jcjdataselection** [comment]
> Hi! Thanks for your interest in our work.
> 
> Based on previous issue reports, the result could be different depending on the FAISS version.
> 
> BTW, as analysed in our paper's appendix, we observed that samples from VQA tasks (e.g., VQAv2, GQA, OKVQA ...) were majorly selected.
> 
> If you let us know your email, we would like to send you the indices for your reference. Please understand that we only have the indices of a 20% pruning ratio for the LLaVA instruction tuning dataset.

 Thanks very much for your reply. I referred to previous issues, and my faiss-gpu version is 1.8.0. My clustering results are also similar to those you provided in other issues. The difference is that I used indices to select a subset from the complete dataset and performed instruction fine-tuning using the official LLaVA code. Finally, my email address is jcj34618@gmail.com.

**Comment by ShristiDasBiswas** [comment]
Hi, running the COINCIDE clustering code generates the following pruned_indices.npy file for me (https://drive.google.com/file/d/1Sn8A2zPA52ndXBGDFBVarqtg7SsZD6rO/view?usp=sharing). Finetuning on this generates a score of 74.67% on vqav2 and only 38% on vizwiz, which is significantly lesser. Would greatly appreciate some pointers on where things could be going wrong if you have faced a similar accuracy drop. My email is shristidasbiswas@gmail.com. Thanks!
